framework/include/fviks/FVInterfaceKernel.h

```diff
@@ -133,13 +137,16 @@ class FVInterfaceKernel : public MooseObject,
   /**
    * Process the provided residual given \p var_num and whether this is on the neighbor side
    */
-  void addResidual(Real resid, unsigned int var_num, bool neighbor);
+  void addResidual(Real resid, unsigned int var_num, Assembly & assembly, bool neighbor);
```

Comment:
Currently, the user creates 2 objects in the input file with var1<->var2 switch. This has issues with some of the more advanced interfaces so I'll have to change it. I also encountered an issue with this with MCRE and are still working on it.  A better implementation is coming soon, I just thought this would work and the 1D test cases I used looked good.

Reply:
I'm splitting the FVInterfaceKernel-based work into another PR, resolving this comments.